docs/reference/database-migration.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Database Migration
+description: A guide to migrating lakeFS database.
+parent: Reference
+nav_order: 51
+has_children: false
+---
+
+# lakeFS Database Migrate
+{: .no_toc }
+
+**Note:** Feature in development
+{: .note }
+
+The lakeFS database migration tool simplifies switching from one database implementation to another.
```

Comment:
I would add an example to make it clear.
```suggestion
The lakeFS database migration tool simplifies switching from one database implementation to another, e.g. from Postgres to DynamoDB.
```

Reply:
This one is coming from the migration PR. It is no longer part of this PR as I changed the base